sdk/src/utils/errorHandler.ts

```diff
@@ -5,12 +5,15 @@ export function handleError(type: ActionType, err: unknown): never {
   if (err instanceof BaseError) {
     const revertError = err.walk((err) => err instanceof ContractFunctionRevertedError);
     if (revertError instanceof ContractFunctionRevertedError) {
-      const errorName = revertError.data?.errorName ?? "";
+      const errorName = revertError.data?.errorName ?? revertError.signature ?? "unknown revert reason";
       throw new Error(`${type} failed with ${errorName}`);
+    } else {
+      const shortMessage = err.shortMessage ?? "An unknown error occurred";
```

Comment:
```suggestion
      const shortMessage = err.shortMessage ?? "an unknown error";
```

To make the default error message smoother 🙂

Reply:
done.